### DISCUSSION TITLE
How should the home page be?

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#20] [Q&A]
How should I sort projects on the homepage?

**Comment by MilesWK** [comment]
Hmm... what do you mean by "sort?"

**Reply by EnZon3** [reply]
Like how projects would be laid out on the homepage.

**Comment by MilesWK** [comment]
I think you should do it like Replit does it or like Scratch does it. Both
are good options in my opinion!

**Reply by EnZon3** [reply]
Kinda forgot how Replit did it tbh

**Comment by MilesWK** [comment]
By putting 6 of the most popular projects at the top, followed by a list of
shared projects and loved projects by people you follow!

**Reply by EnZon3** [reply]
K!

**Reply by EnZon3** [reply]
I'll try to see if I can get some prototype running in my space.
